druid-shell/src/platform/x11/runloop.rs

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The xi-editor Authors.
```

Comment:
I'm very confused, this should have been deleted 😖

Reply:
Lol -- updated all files in `druid-shell/src/platform/x11` to 2020. Unless you wanted all of those headers deleted, in which case I could do that too.

Reply:
Oh, you meant the entire `runloop.rs` file. Hmmm

Reply:
Ok, yep, I was able to just delete it!